python/tvm/relay/prelude.py

```diff
@@ -482,8 +482,8 @@ def load_prelude(self):
         with open(prelude_file) as prelude:
             prelude = fromtext(prelude.read())
             self.mod.update(prelude)
-            self.id = self.mod["id"]
-            self.compose = self.mod["compose"]
+            self.id = self.mod.get_global_var("id")
+            self.compose = self.mod.get_global_var("compose")
```

Comment:
aah... why?

Reply:
`mod["id"]` returns `Function`. But the function name symbols need to be `GlobalVar`, you can check other prelude functions. Otherwise it would cause errors like https://discuss.tvm.ai/t/relay-to-a-normal-form-fails-on-a-prelude-function/3031.

Reply:
I see.